src/main/java/com/synopsys/integration/alert/web/user/RoleActions.java

```diff
@@ -53,7 +51,6 @@
 import com.synopsys.integration.alert.web.model.RolePermissionModel;
 
 @Component
-@Transactional
```

Comment:
Deletes wouldn't work because the delete uses this transaction.  The accessor deletes the entity but then with the loadPermissionIntoCache calls a get from the accessor and PSQL complains because we are not in a read only transaction.  Use the transactions in the accessor and remove transactional here.